Editor/GitDiffWindow.cs

```diff
@@ -57,25 +57,45 @@ public class GitDiffWindow : DefaultWindow
 
         public static LazyStyle lineNumber = new(() => new()
         {
-            fontSize = 10
+            fontSize = 10,
+            normal = new GUIStyleState()
+            {
+                textColor = Style.TextColor
```

Comment:
It would be nice to have a root/parent style to assign the default `textColor` since it is assigned too many times.

Reply:
I've added an issue